thefuck/rules/terraform_no_command.py

```diff
@@ -0,0 +1,16 @@
+import re
+from thefuck.utils import for_app
+
+MISTAKE = r"(?<=Terraform has no command named \")(.*)(?=\"\.)"
+FIX = r"(?<=Did you mean \")(.*)(?=\"\?)"
```

Comment:
How about being a bit more restrictive and remove some unnecessary backslashes?

```suggestion
MISTAKE = r'(?<=Terraform has no command named ")([^"]+)(?="\.)'
FIX = r'(?<=Did you mean ")([^"]+)(?="\?)'
```